addons/flannel/0.20.2/install.sh

```diff
@@ -26,6 +26,11 @@ function flannel_pre_init() {
     flannel_init_pod_subnet
 }
 
+function flannel_join() {
+    logWarn "Flannel requires UDP port 8472 for communication between nodes."
+    logWarn "Failure to open this port will cause connection failures between containers on different nodes."
```

Comment:
This language is meant to suggest that the user make sure that the port is open. Does this wording make it seem that the port is indeed not open? If so please suggest a copy edit.